src/Vipps.net/Services/AccessTokenCacheService.cs

```diff
@@ -7,22 +7,28 @@
 
 namespace Vipps.net.Services
 {
-    public static class AccessTokenCacheService
+    public class AccessTokenCacheService
     {
 #pragma warning disable IDE0090 // Use 'new(...)'
```

Comment:
I think you can remove the pragmas now, since the `new` calls now are placed in a constructor